truss/templates/control/proxy.conf

```diff
@@ -0,0 +1,28 @@
+
+server {
+    listen 8080;
+
+    # Liveness
+    location / {
+        add_header Content-Type text/plain;
+        return 200 "";
+    }
+
+    # Readiness
+    location ~ ^/v1/ {
+        if (!-f "~/inference_server_crashed.txt") {
```

Comment:
we shouldn't use the tilde symbol here, instead let's use the fully qualified path instead. (Same thing goes for the Truss code). Reason: i'm not positive if the tilde resolves correctly in all contexts (this is not bash), and using fully qualified path reduces ambiguity

I just read through this doc: https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/, I don't think we can get away from using `if` here though.

Reply:
Happy to move this, but to do that, we have to commit to not allowing the draft server to run without docker. The path in the docker image is predictable and permitted. on a user's system, we would have a problem. I am personally okay to make this commitment, but would need a 👍  from PG on that move?

Reply:
to be honest, I think with this change it's already going to be very hard to run the draft server without docker. ie: user will need to have nginx and superivisord to run it (and the right versions furthermore). I don't expect to people to be able to do this reliably.